src/templates/static/legal/risk_disclaimer.jsx

```diff
@@ -2,59 +2,64 @@ import React from 'react';
+        <h2 data-anchor='internet-trading-risks'>{it.L('Internet trading risks')}</h2>
+        <p>{it.L('There are risks associated with utilising an Internet-based deal execution trading system, including, but not limited to, the failure of hardware, software and Internet connection. Since [_1] does not control signal power, its reception or routing via Internet, the configuration of the client\'s equipment, or the reliability of its connection, the Company cannot be responsible for communication failures, distortions, or delays during online trading.', it.website_name)}</p>
 
-        <h2 data-anchor='accuracy-of-information'>{it.L('Accuracy of Information')}</h2>
+        <h2 data-anchor='accuracy-of-information'>{it.L('Accuracy of information')}</h2>
         <p>{it.L('The content on this website is subject to change at any time without notice and is provided for the sole purpose of assisting traders in making independent investment decisions. [_1] has taken reasonable measures to ensure the accuracy of the information on the website.', it.website_name)}</p>
 
-        <h2 data-anchor='costs-and-charges'>{it.L('Costs and Charges')}</h2>
-        <p>{it.L('Our costs and charges for depositing and withdrawing money are set out [_1]here[_2]. Please be aware of all costs and charges that apply to you because they will affect your profitability.', `<a href="${it.url_for('cashier/payment_methods')}">`, '</a>')}</p>
+        <h2 data-anchor='costs-and-charges'>{it.L('Costs and charges')}</h2>
+        <p>{it.L('The Company\'s costs and charges for depositing and withdrawing money are set out [_1]here[_2]. The client should be aware of all costs and charges that apply to the client because they will affect the client\'s profitability.', `<a href="${it.url_for('cashier/payment_methods')}">`, '</a>')}</p>
+
+        <h2 data-anchor='cryptocurrency-risks'>{it.L('Cryptocurrency risks')}</h2>
+        <p>{it.L('Cryptocurrencies are volatile assets that are extremely vulnerable to market fluctuations. Trading in cryptocurrencies is a highly speculative investment because prices of cryptocurrencies can go up and down — potentially to a worthless (zero) value. The client should not trade in cryptocurrencies unless the client fully understands and is aware that trades and/or investments involving cryptocurrencies are complex and high risk in nature, and the client not only risk incurring significant losses over a short period of time but also risk losing all the client\'s invested capital.')}</p>
```

Comment:
```suggestion
        <p>{it.L('Cryptocurrencies are volatile assets that are extremely vulnerable to market fluctuations. Trading in cryptocurrencies is a highly speculative investment because prices of cryptocurrencies can go up and down — potentially to a worthless (zero) value. The client should not trade in cryptocurrencies unless the client fully understands and is aware that trades and/or investments involving cryptocurrencies are complex and high risk in nature, and the client not only risks incurring significant losses over a short period of time but also risks losing all the client\'s invested capital.')}</p>
```